src/main/java/com/yash/ExpenseClaims/controllers/RoleController.java

```diff
@@ -0,0 +1,44 @@
+package com.yash.ExpenseClaims.controllers;
+
+import com.yash.ExpenseClaims.dto.RoleDto;
+import com.yash.ExpenseClaims.services.RoleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.*;
+
+@RestController
+public class RoleController {
+
+    @Autowired
+    private RoleService roleService;
+
+    @RequestMapping(value = "/roles", method = POST)
+    public RoleDto createRole(@RequestBody RoleDto roleDto) {
+        return roleService.createRole(roleDto.getRole_name());
```

Comment:
:x: not an industry standard to use underscores in method names, especially with getters/setters

Reply:
We realised this java file is not necessary anymore. So removed it.